src/app/app.component.ts

```diff
@@ -57,6 +57,7 @@ export class AppComponent implements OnInit {
 			"gr",
             "te",
             "en",
+            "ka",
```

Comment:
Can you move it up? English needs to be the last one on this list.

Reply:
Yeah sure!